src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -491,3 +494,30 @@ registerAction2(class extends Action2 {
 	}
 });
 
+// peekView.border
+registerColor('interactive.activeCodeBorder', {
+	dark: '#007acc',
+	light: '#007acc',
```

Comment:
I think these should pull from `focusBorder`? Otherwise they won't be themeable

Reply:
IIRC this was inspired by peekView.border so I added `registerThemingParticipant` and pulled `theme.getColor(peekViewBorder)`, LMK if I should have done this a different way.